src/SimpleThings/EntityAudit/AuditReader.php

```diff
@@ -748,12 +748,24 @@ public function getEntityValues($className, $entity)
     {
         $metadata = $this->em->getClassMetadata($className);
         $fields = $metadata->getFieldNames();
+        $associationsFields = $metadata->getAssociationNames();
 
         $return = array();
         foreach ($fields AS $fieldName) {
             $return[$fieldName] = $metadata->getFieldValue($entity, $fieldName);
         }
 
+        foreach($associationsFields as $key => $association) {
+            $associationValues = $entity->{'get' . ucfirst($association)}();
```

Comment:
We shouldn't use getters. You don't have it for each property (for example if you develop DDD). We should diff over properties and use `$metadata->getFieldValue($entity, $fieldName)` for it.